lib/cli/src/generators/configure.ts

```diff
@@ -19,6 +19,12 @@ function configurePreview(framework: SupportedFrameworks) {
   const parameters = `
 export const parameters = {
   actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
```

Comment:
we could maybe add a link to the docs as a comment?

Reply:
great idea, let's follow up when the new docs are live